apps/web/vibes/soul/sections/header/index.tsx

```diff
@@ -147,7 +158,7 @@ export const Header = forwardRef(function Header(
     const searchTerm = searchInputRef.current?.value.trim() ?? ''
 
     if (searchTerm.length > 0) {
-      router.push(`/search?term=${encodeURIComponent(searchTerm)}`)
+      router.push(`/${searchHref}?term=${encodeURIComponent(searchTerm)}`)
```

Comment:
Minor nit on the naming here. If we're naming it "href" it should probably be the full  relative or absolute URL, no? If so, we don't want to pre-pend the leading `/`? Also, if the provided URL already has search params then hardcoding `?` might give unexpected results.

Reply:
Also, it seems we pass `accountHref` to an `href` attribute but for this one we do some more custom stuff. Might not be relevant for this PR but perhaps we want to handle this with a different approach?

Reply:
Total oversight on my part, href should include `/` like you mention.

I think this is a simple form action that would use the new form component in Next 15. All we want to do is trigger a redirect to `/search?term=blah`. I don't think we should care about passing in any other search params with this redirect, it's not commonly a thing afaik.

Reply:
Yeah. Do you want to take care of using the new form component in this PR or leave is as a todo for a future PR? My guess is we'll come back to this header component and do another pass before 11/11.

Reply:
Lets solve in a different PR for a different time.